extensions/components/com_redcore/site/tables/translation.php

```diff
@@ -0,0 +1,146 @@
+<?php
```

Comment:
this file is not needed as it will load table from admin path, not site. I did not checked if you have changed anything here.